monkey/monkey_island/cc/services/reporting/exploitations/monkey_exploitation.py

```diff
@@ -36,28 +41,32 @@ def get_monkey_exploited(
             label=str(machine.network_interfaces[0].ip),
             ip_addresses=[str(iface.ip) for iface in machine.network_interfaces],
             domain_name="",
-            exploits=get_exploits_used_on_node(machine, successful_exploits),
+            exploits=get_exploits_used_on_node(machine, successful_exploits, plugin_manifests),
         )
         for machine in exploited_machines
     ]
 
-    logger.info("Exploited nodes generated for reporting")
+    logger.info(f"Exploited nodes generated for reporting {exploited}")
```

Comment:
I don't think this is a valuable addition to the loggs

Reply:
Forgot it :)